modules/renter/download.go

```diff
@@ -112,9 +113,18 @@ func (d *Download) start() {
 		for _, piece := range d.pieces {
 			downloadErr := d.downloadPiece(piece)
 			if downloadErr == nil {
-				d.complete = true
-				d.file.Close()
-				return
+				// Decrypt the file.
+				d.file.Seek(0, 0)
+				cryptBytes, err := ioutil.ReadAll(d.file)
```

Comment:
this is a short term solution. The longer term solution involves loading about 10MB at a time and leveraging micro-diffs to keep all of the memory requirements sane.